### DISCUSSION TITLE
Thousand Comma Separated Value

### DISCUSSION THREAD

**Discussion by RanggaGemilang** [teamalgoritma/community#185] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107049922/178097123-59b5ce28-5562-4a92-b7be-04e46f17ece5.png)

How can I show these values from **Volume** with a thousand separator comma so the example value will be 10,000,000,000 **while** maintaining its integer dtype. Thank you :)

**Comment by yosiaazarya** [comment, marked as answer]
Hi! To separate thousands in pandas, we can do it in several ways:
1. With `df = df.style.format({'Volume($'): "{:,}"})` 
2. `df['Volume($')'] = df['Volume($')'].map("{:,}".format)`
3. `pd.options.display.float_format = '{:,}'.format`

I hope it could help! :)

**Reply by RanggaGemilang** [reply]
![Screenshot 2022-07-11 130327](https://user-images.githubusercontent.com/107049922/178198923-a8cc9fb2-c88b-4dfa-bfe4-e67d15b625bd.png)
Hi mas yosia, thanks for the answer. :)

However, after I tried the third code, the value is still looks like this. As the first and second, the value displays as intended but the data types changed into 'object' instead. 

Is there anyway to maintain the 'int' data types while the value shows as intended?
